hydra-node/golden/ReasonablySized (ServerOutput SimpleTx).json

```diff
@@ -2236,9 +2271,11 @@
             ]
         },
         {
+            "headStatus": "Idle",
             "me": {
-                "vkey": "bca9150f8ede476548571de3e4dcad31a620f54085b307ebbbb95b3003684148"
+                "vkey": "05446c191ec3f312872da337aa59c92325b28de4ae150419838673f5e8396838"
             },
+            "snapshotUtxo": [],
```

Comment:
Should: Not serialize empty array here when there are no utxo.

This is a nice example of why this looks unintuitive on the API.